src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -554,7 +556,8 @@ function MoneyRequestConfirmationList(props) {
                 success
                 pressOnEnter
                 isDisabled={shouldDisableButton}
-                onPress={(_event, value) => confirm(value)}
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                onPress={(_event, value) => confirm(value as PaymentMethodType)}
                 options={splitOrRequestOptions}
```

Comment:
Since ButtonWithDropdownMenu has `genric` in typing, you can try to update `splitOrRequestOptions` typing to include `PaymentMethodType` instead of `string`. Though `iouType` and `PaymentMethodType` are not aligned, so there will be another TS problem.

Reply:
I ended up typing `confirm` fn to by `IouType | PaymentMethodType | undefined` since I check and here from `value` we can only `IouType` based on implementation of `splitOrRequestOptions` but looking and implementation of `sendMoney` which is called inside `confirm`, `sendMoney` expects that the type will be `PaymentMethodType` which is odd since from my understanding it will never happen here